### PR TITLE
- developed a webpage for todoList.

### PR DESCRIPTION
**Pull request by snehapoulose682**
Developed a UI design for To-do-List using HTML and CSS

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    padding: 12px 10px 12px 40px;
+    background-color: whitesmoke;
+    border-bottom: 1px solid black;
+}
+ .app ul li i {
+    font-size: 30px;
+    float: right;
+    cursor: pointer;
+}
+
```
Comment (dheeraj-murali) [inline style.css]: Style items based on class names, rather than the element it self

File: style.css [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    padding-top: 50px;
+}
+.app {
+    width: 60%;
+}
+@media (max-width:800px){
+    .app {
+        width: 90%;
+    }
+}
```
Comment (dheeraj-murali) [inline style.css]: Loved that you have used media queries to build a responsive app. but make sure to take a mobile first approach to build web UIs

File: index.html [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+        <div class="app">
+            <h2>Todo List</h2>
+            <form>
+                <input type="text" placeholder="Enter your task">
+                <button type="submit">Add</button>
+            </form>
+            <ul>
+                <li><input type="checkbox">Item 1 <i class="fa fa-trash"></i></li>
+                <li><input type="checkbox">Item 1 <i class="fa fa-trash"></i> </li>
+                <li><input type="checkbox">Item 1 <i class="fa fa-trash"></i></li>
```
Comment (dheeraj-murali) [inline index.html]: Just having input type as checkbox will not be enough for this to work. more on that here: 
- https://www.w3schools.com/tags/att_input_type_checkbox.asp
- https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox

**Review by dheeraj-murali — CHANGES_REQUESTED** [review]
Good job. few things to be updated

File: style.css (hunk 1 above)
Comment (snehapoulose682) [reply style.css]: Added  class names and removed element names.

File: index.html (hunk 3 above)
Comment (snehapoulose682) [reply index.html]: Added new features to the checkbox.

**Pull request merged**